formula_renames.json

```diff
@@ -36,6 +36,7 @@
   "eigen32": "eigen@3.2",
   "elasticsearch17": "elasticsearch@1.7",
   "elasticsearch24": "elasticsearch@2.4",
+  "elasticsearch502": "elasticsearch@5.0.2",
```

Comment:
This is not necessary as you are not renaming a formula previously called 'elasticsearch502'.